nvflare/apis/job_def_manager_spec.py

```diff
@@ -152,6 +152,20 @@ def get_client_data(self, jid: str, client_name: str, data_type: str, fl_ctx: FL
         """
         pass
 
+    @abstractmethod
+    def list_client_data(self, jid: str, fl_ctx: FLContext):
+        """Get list of all the client data for the specified job.
+
+        Args:
+            jid (str): Job ID
+            fl_ctx (FLContext): FLContext information
+
+        Returns:
+            data content
+
+        """
```

Comment:
What do we mean by "all the client data for the specified job"
Do we mean "all the data coming from all the client sites that associates with this job"?
One example being all the client job logs?

Could we be more specific for the return, by data content do we expect bytes? or a list of names?